internal/conditions/cerbos_lib.go

```diff
@@ -104,16 +113,6 @@ func (clib cerbosLib) ProgramOptions() []cel.ProgramOption {
 				Binary:   callInStringStringOutBool(clib.inIPAddrRangeFunc),
 			},
 
-			&functions.Overload{
-				Operator: timeSinceFn,
```

Comment:
Previously there were two overloads defined here, `timeSince` and `timeSince_timestamp` - I don't think this was necessary because only the latter one was registered in the `decls.NewFunction` call above. I've simplified to just have one overload, registered with the function name (matching the other functions).

Reply:
Yeah, I think that was a leftover from a botched refactoring. Good catch!